Completion.py

```diff
@@ -111,14 +139,12 @@ def read_settings():
         settings["replace_file_path"] = (replace["from"], replace["to"])
     return settings
 
-
-def is_cpp(view):
-    '''Determine if the given view location is c++ code'''
-    try:
-        return view.match_selector(view.sel()[0].begin(), 'source.c++')
-    except:
-        return False
-
+def lang(view):
+    languages = read_settings()['languages']
```

Comment:
I think, that enabling of new language would be a very rare operation. So we can cache this value and add one more sublime command to reload active languages. 
So workflow would be this:
1) Add new language to list in settings
2) Call reload command in sublime command palette